### PR TITLE
feat: SAAA-5875 Add youbora plugin for android

### PR DESCRIPTION
**Pull request by sharan-SKY**
Youbora plugin integration in android.

### CODE REVIEW

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 1]
```diff
@@ -394,6 +402,46 @@ private void initializePlayer() {
         new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
+                YouboraLog.setDebugLevel(YouboraLog.Level.VERBOSE);
+
+                Options youboraOptions = new Options();
+                youboraOptions.setAccountCode(analyticsMeta.getString("accountCode"));
```
Comment (sharan-SKY) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:408]: Unfortunately calling set methods explicitly seems to be the only way for youbora to start recognising that the exoplayer is hooked up to it. These set methods have been used as per the confluence page

https://skynz.atlassian.net/wiki/spaces/SO/pages/5571129/Youbora+Implementation+-+Set+Youbora+Options?focusedCommentId=1565818922

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java [hunk 2]
```diff
@@ -394,6 +402,46 @@ private void initializePlayer() {
         new Handler().postDelayed(new Runnable() {
             @Override
             public void run() {
+                YouboraLog.setDebugLevel(YouboraLog.Level.VERBOSE);
```
Comment (serge-sky) [inline android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java:405]: we want to disable it for release

**Review by serge-sky — APPROVED** [review]
looks great, well done 👍

File: android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java (hunk 2 above)
Comment (sharan-SKY) [reply android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java]: I have resolved this

**Pull request merged**